app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,345 @@
+* [Create API product](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0#/API%20Products/create-api-product)
+* [Create API product version specification](https://developer.konghq.com/spec/d420333f-01b0-464e-a87a-97acc92c2026/941af975-8cfa-40f7-afea-e82d248489a0#/API%20Product%20Version%20Specification/create-api-product-version-spec)
+
+Review the API documentation to see what other features are available.
+{% endnavtab %}
+{% endnavtabs %}
+
+### Create developer teams
+
+Next, let's create two developer teams: "Authorized Delivery Partners" with API Viewer and Consumer permissions and "Prospective Partners" with API Consumer permissions.
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon dev-portal %} **Dev Portal**.
+1. Click **Teams** in the sidebar and then click **Add team**.
+1. Enter the following:
+    * **Team name:** Authorized Delivery Partners
+    * **Description:** Team of vetted delivery partners that can view and consume APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a test developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Pizza Ordering** product from the dropdown menu and select **API Consumer** and **API Viewer** from the **Add roles** dropdown menu.
+1. Click **Teams** in the sidebar and click **New Team** again and enter the following:
+    * **Team name:** Prospective Partners
+    * **Description:** Team of unvetted partners that can only view APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a test developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Pizza Ordering** product from the dropdown menu and select **API Viewer** from the **Add roles** dropdown menu. 
+
+You now have added your test developer accounts to two teams with different access permissions for your Dev Portal. These developers won't be able to access anything in Dev Portal yet, though, because we need to enable Portal RBAC next.
+{% endnavtab %}
+{% navtab API %}
+The {{site.konnect_short_name}} API uses [Personal Access Token (PAT)](/konnect/api/#authentication) authentication. You can obtain your PAT from the [personal access token page](https://cloud.konghq.com/global/account/tokens). The PAT must be passed in the `Authorization` header of all requests.
+
+1. Create the `Authorized Delivery Partners` team:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/{portal-id}/teams \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --header 'accept: application/json' \
+  --data "name": "Authorized Delivery Partners" \
+  --data "description": "Team of vetted delivery partners that can view and consume APIs"
+```
```

Comment:
this call did not work for me as written now. this did work:

`curl --request POST \
  --url https://us.api.konghq.com/v2/portals/{portal-id}/teams \
  --header 'Authorization: Bearer <personal-access-token>' \
  --header 'Content-Type: application/json' \
  --data '{"name": "Authorized Delivery Partners","description": "Team of vetted delivery partners that can view and consume APIs"}'`